resume.tex

```diff
@@ -82,7 +79,7 @@
       Node.js,
       AngularJS (1.X),
       Angular (2+),
-	  jQuery,
+      jQuery,
       Jasmine,
       Bootstrap%,
       %React %(familiar)%,
```

Comment:
I agree with omitting the "(familiar)" ones, because as they always warn, things on your resume are fair game for nasty interviewers.

Reply:
Mmhmm, generally I'll only uncomment these if it's particularly relevant to a job description.  If a company builds their client applications in React, it seems like it would be good to show that I know the basics, even though I haven't regularly used it at work.